apps/gallery/style/gallery.css

```diff
@@ -41,6 +41,10 @@ section {
   display: none;
 }
 
+body.showing-dialog > section {
```

Comment:
I'll add a comment  about that in the CSS. The screen reader does not distinguished between obscured and un-obscured content. First, because it is really hard to do, you need to take into account the relative opacity of each obscuring layout frame, and false positives would be very bad. Also, historically developers would hide text intended for screen readers in different fashions, like putting it in a negative offset, using font-size:0 or obscuring it. The issue of modal dialogs is not unique to this module, and is the most painful part of making gaia accessible. We will need to do this until the <dialog> element is implemented in gecko.